src/oop/app/src/Parsers/FilmixParserStrategy.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace src\oop\app\src\Parsers;
+
+use src\oop\app\src\Models\Movie;
+
+class FilmixParserStrategy implements ParserInterface
+{
+    private Movie $movie;
+
+    /**
+     * Generating Movie class just after class init
+     */
+    public function __construct()
+    {
+        $this->movie = new Movie();
```

Comment:
please do not use new keyword, use dependency injection:
public function __construct(Movie $movie)
{
$this->movie= $movie;

Reply:
> please do not use new keyword, use dependency injection: public function __construct(Movie $movie) { $this->movie= $movie;

In this case, I'd modify ScrapperFactory.php file and pass there a Movie object, am I right?
I am not sure if it's allowed. 
Also, I do not understand what is the point - does it have any sense to throw (empty) objects through different classes?

Reply:
please read about SOLID and Dependency Injection